polling_stations/templates/base.html

```diff
@@ -38,7 +38,7 @@
             <meta name="twitter:image" content="{% block og_image_twitter_content %}{% static 'images/logo_icon.png' %}{% endblock og_image_twitter_content %}" />
             <meta name="twitter:description" content="{% block og_twitter_description %}{% if NEXT_CHARISMATIC_ELECTION_DATE %}{% blocktrans with election_date=NEXT_CHARISMATIC_ELECTION_DATE|date:'F j Y' %}Find out where to vote on {{ election_date }}{% endblocktrans %}{% else %}{% trans "Find out where to vote" %}{% endif %}{% endblock og_twitter_description %}">
             <meta name="twitter:image:alt" content="{{ SITE_TITLE }}">
-            <link rel="shortcut icon" href="{%  static 'images/logo_icon.svg' %}">
+            <link rel="shortcut icon" href="{% static 'images/logo_icon.png' %}">
```

Comment:
I'm not sure what is being fixed here. Previously for me this introduces a white box around the DC logo circle
svg:
![SVG version](https://user-images.githubusercontent.com/20044500/153166929-625ff224-f295-4a44-b056-668b600ace97.png)
png:
![PNG Version](https://user-images.githubusercontent.com/20044500/153166945-636382b0-3a33-4c1d-b94d-c09d2aeb5aa3.png)

There maybe other benefits to the png over svg, but it would be good to understand what they are.

Reply:
Actually, the filetype wasn't the problem after all. Made a change that should provide wider browser support.

Reply:
`.png` has wider browser support over `.svg` and then there is `.ico` for IE.